app/modules/users/components/users_home_section_list.svelte

```diff
@@ -1,17 +1,21 @@
 <script>
+  import { BubbleUpComponentEvent } from '#lib/svelte/svelte'
   import { i18n } from '#user/lib/i18n'
   import UsersHomeSectionListLi from '#users/components/users_home_section_list_li.svelte'
+  import { createEventDispatcher } from 'svelte'
 
   export let docs, type, hideList = false, hideListMessage
+  const dispatch = createEventDispatcher()
+  const bubbleUpComponentEvent = BubbleUpComponentEvent(dispatch)
 </script>
 
 {#if hideList}
   <p class="hidden-list-message">{hideListMessage}</p>
 {:else}
-  <ul class:members={type === 'members'}>
+  <ul class:members={type === 'members'} class="users-home-section-list">
     {#each docs as doc (doc._id)}
       <li class="user">
-        <UsersHomeSectionListLi {doc} />
+        <UsersHomeSectionListLi {doc} on:select={bubbleUpComponentEvent} />
```

Comment:
i guess i let through during the `users-layout-rewrite` review, but the `CurrentEvents` created by `createEventDispatcher`[dont bubble](https://svelte.dev/docs#run-time-svelte-createeventdispatcher) per se (aka to the parent DOM component), here BublingUp refers to the parent Svelte component not the DOM, right? Hence `dispatch` sounds more appropriate?
```suggestion
        <UsersHomeSectionListLi {doc} on:select={dispatchEvent} />
```

Reply:
But what is the diff between bubble up and dispatch? If no difference, why not sticking to svelte naming? (+ fewer keystrokes).

Reply:
My understanding of the nuance is that `dispatch` triggers an event on an object ([typically a DOM element in web standard](https://dom.spec.whatwg.org/#events)), while the `bubble` flag determines if that event is automatically forwarded to parent elements of the DOM. Dispatching and forwarding are different in the fact that dispatching sets the object as the event origin (typically the `event.currentTarget` in DOM events), while forwarding just passes the event up without changing the event origin(?). [Svelte tutorial uses a function named `forward`](https://svelte.dev/tutorial/event-forwarding), maybe that's a better name for that function?